tests/PySys/software-management-end-to-end/environment_sm_management.py

```diff
@@ -56,10 +90,36 @@ def setup(self):
         if self.myPlatform != "specialcontainer":
             self.skipTest("Testing the apt plugin is not supported on this platform")
 
+        # Database with package IDs taken from the thin-edge.io
+        # TODO make this somehow not hard-coded
+        self.pkgiddb = {
```

Comment:
```suggestion
        self.pkg_id_db = {
```
Would have been nice if we could use snake case everywhere. Currently it's a mix of snake case, camel case and case-less names like these ;)